src/test/java/org/jenkinsci/plugins/kubernetes/cli/KubectlBuildWrapperTest.java

```diff
@@ -140,10 +142,10 @@ public void testListingCredentialsWithoutAncestorAndMissingPermissions() throws
 
         r.jenkins.setSecurityRealm(r.createDummySecurityRealm());
         ProjectMatrixAuthorizationStrategy as = new ProjectMatrixAuthorizationStrategy();
-        as.add(Jenkins.READ, "user-not-enough-permissions");
+        as.add(Jenkins.READ, new PermissionEntry(AuthorizationType.EITHER, "user-not-enough-permissions"));
```

Comment:
For myself: 
![image](https://github.com/jenkinsci/kubernetes-cli-plugin/assets/3045354/f3a6ba13-3277-448c-943c-3dfb15e47e56)